.github/actions/build-deb-package/action.yml

```diff
@@ -18,26 +18,31 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Check out repository
-      uses: actions/checkout@v4
-
     - name: Build Docker Image
-      run: docker build -f ${{ inputs.dockerfile_path }} -t ${{ inputs.docker_image }} .
+      run: docker build --pull -f ${{ inputs.dockerfile_path }} -t ice-deb-package-builder .
       shell: bash
 
     - name: Run Package Build with Mounted Source
       run: |
-        mkdir -p output
+        mkdir -p "$HOME/build"
```

Comment:
I think it would be cleaner to clone ice into `${GITHUB_WORKSPACE}/ice` like we do for the demos and create `${GITHUB_WORKSPACE}/build` for the build to use

Reply:
Fixed as suggested